google/cloud/spanner_dbapi/connection.py

```diff
@@ -540,7 +550,8 @@ def partition_query(
         return partition_ids
 
     @check_not_closed
-    def run_partition(self, encoded_partition_id):
+    def run_partition(self, parsed_statement: ParsedStatement):
```

Comment:
This is a breaking change of a public method

Reply:
Reverted